src/options.ts

```diff
@@ -1081,6 +1083,7 @@ const options = mkOptions(CONFIG, {
                 updateCommand: opt(`${SRC_DIR}/scripts/checkUpdates.sh -arch`),
                 label: opt(true),
                 padZero: opt(true),
+                autoHide: opt(true),
```

Comment:
```suggestion
                autoHide: opt(false),
```
We should leave it disabled by default.